src/main.js

```diff
@@ -118,14 +116,14 @@ function updateCircleColor(button, classProperty) {
 function createCurrentActivity() {
```

Comment:
Maybe rename this function? perhaps "displayCurrentActivitySection"?

Reply:
If you guys are worried about have naming conventions too long dont. It better to be as clear as possible. Think about developer empathy. Think what somebody that has never seen your code before and how long it would take them to figure out what each function does.